src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/PtvConsts.java

```diff
@@ -13,6 +13,7 @@
   public static final String SYSTEM_SETTING_BASEURL = "ptv.baseUrl";
   public static final String DEFAULT_LANGUAGE = "fi";
   public static final String TIMEZONE = "Europe/Helsinki";
+  public static final String[] PTV_SUPPORTED_LANGUAGES = {"fi", "en", "sv"};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2386)